src/NuGetForUnity/Editor/Ui/NugetWindow.cs

```diff
@@ -934,44 +967,71 @@ private void DrawPackage(INugetPackage package, GUIStyle packageStyle, GUIStyle
+                    if (package.Versions.Count <= 1)
                     {
-                        EditorStyles.popup.CalcMinMaxWidth(
-                            new GUIContent(
-                                package.Versions.Select(version => version.FullVersion).OrderByDescending(version => version.Length).First()),
-                            out _,
-                            out var maxWidth);
-                        versionDropdownData = new VersionDropdownData
-                        {
-                            SortedVersions = package.Versions.OrderByDescending(version => version).ToList(), CalculatedMaxWith = maxWidth + 5,
-                        };
-                        versionDropdownData.DropdownOptions = versionDropdownData.SortedVersions.Select(version => version.FullVersion).ToArray();
-                        versionDropdownData.SelectedIndex = versionDropdownData.SortedVersions.IndexOf(package.PackageVersion);
-                        versionDropdownDataPerPackage.Add(package.Id, versionDropdownData);
+                        GUILayout.Label($"Version {package.PackageVersion.FullVersion}");
                     }
+                    else
+                    {
+                        if (!versionDropdownDataPerPackage.TryGetValue(package.Id, out var versionDropdownData))
+                        {
+                            EditorStyles.popup.CalcMinMaxWidth(
+                                new GUIContent(
+                                    package.Versions.Select(version => version.FullVersion).OrderByDescending(version => version.Length).First()),
+                                out _,
+                                out var maxWidth);
+
+                            if (installed != null)
+                            {
+                                // we're showing only lower or only higher versions, depending on the 'Show Downgrades' checkbox.
+                                // This can only happen in Updates tab.
+                                if (showDowngrades)
+                                {
+                                    package.Versions.RemoveAll(version => version >= installed.PackageVersion);
+                                }
+                                else
+                                {
+                                    package.Versions.RemoveAll(version => version <= installed.PackageVersion);
+                                }
+                            }
+
+                            versionDropdownData = new VersionDropdownData { SortedVersions = package.Versions, CalculatedMaxWith = maxWidth + 5, };
+
+                            versionDropdownData.DropdownOptions = versionDropdownData.SortedVersions.Select(version => version.FullVersion).ToArray();
+
+                            // Show the highest available update/downgrade first
+                            versionDropdownData.SelectedIndex = 0;
```

Comment:
On the Update / Online tab I would show the highest version, on the installed we can show the current selected. See Visual Studio package window.